blog/prometheus-connector.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Skydive-Prometheus Connector
+author: Kalman Meth
+layout: blog-post
+date: 1/11/2020
+---
+[Prometheus](https://prometheus.io/) is an open-source systems monitoring and alerting tool.
+Many different types of data are collected by different tools and are forwarded to Prometheus via various types of exporters.
+For example, the [Prometheus Node Exporter](https://prometheus.io/docs/guides/node-exporter/) exposes a wide variety of hardware- and kernel-related metrics.
+We developed an exporter for Skydive that reports metrics of individual captured network flows.
+The connector translates data from Skydive captured flows into a format that can be consumed by Prometheus.
+The first implementation of the Skydive-Prometheus connector periodically provides the byte transfer counts for each network connection under observation.
+The code can be easily tailored to provide additional flow information.
+
+The structure of the interacting components is depicted in the following figure.
+
+<p align="center">
+  <a href="/assets/images/blog/SkydivePromethusConnector.png" data-lightbox="prom_sky_con" data-title="Skydive-Prometheus connector">
+    <img src="/assets/images/blog/SkydivePromethusConnector.png" style="width:90%;"/>
+  </a>
+</p>
+
+## Deploy Skydive
+
+You can either download a statically linked version of Skydive or you can build it on your own.
+
+To download a pre-compiled statically linked version of Skydive, run the following command.
+
++{% highlight shell %}
```

Comment:
There are some extra `+` remaining in the file that need to be removed. It seems to be only before jekyll tags.

Reply:
Fixed.